### DISCUSSION TITLE
Why I was block from Discord without contact me?

### DISCUSSION THREAD

**Discussion by juanquy** [ExpressLRS/ExpressLRS#2135] [General]
That was fucking rude!
User = TurboWalksnail#4966

**Comment by JyeSmith** [comment]
This account was banned due to spamming a link to a NSFW discord.

Bit fucking rude hey?

**Comment by juanquy** [comment]
spamming my account?
and yes rude I didn't know my account was spamming, why I should email you
from my personal account if I'm a spammer idiot?
you should contact the person before idiot!
restore my account

**Comment by JyeSmith** [comment]
No.
